### DISCUSSION TITLE
Request: 添付データ一覧に作成日時も表示して欲しい

### DISCUSSION THREAD

**Discussion by sakazuki** [growilabs/growi#8230] [Ideas / Requests]
「画像以外のファイルアップロードを許可」して使用しています。

現状、「添付データ」の一覧（PageAttachmentList）ではファイル名（originalName）が表示されますが、
同名ファイルを複数回アップしていると、ファイル名だけでは区別ができません。

一覧に作成日時を追加で表示してもらえないでしょうか。

PRも上げさせていただきました。https://github.com/weseek/growi/pull/8229
ご確認いただけますと幸いです

**Comment by yuki-takei** [comment]
PR ありがとうございます！確認します

**Comment by sakazuki** [comment]
#8229 をマージしていただきました。ありがとうございます！
